src/Workspaces/Core/Portable/Shared/Extensions/DocumentExtensions.cs

```diff
@@ -23,6 +23,10 @@ internal static partial class DocumentExtensions
         public static TLanguageService GetLanguageService<TLanguageService>(this TextDocument document) where TLanguageService : class, ILanguageService
             => document?.Project?.LanguageServices?.GetService<TLanguageService>();
 
+        // ⚠ Verify IVTs do not use this method before removing it.
```

Comment:
I both love the emoji and fear for what it will break.

Reply:
I've used it in a few other cases and it seems to work great.